crates/compiler/parse/src/state.rs

```diff
@@ -2,7 +2,7 @@ use roc_region::all::{Position, Region};
 use std::fmt;
 
 /// A position in a source file.
-#[derive(Clone)]
+#[derive(Clone, Copy)]
```

Comment:
I think we deliberately removed the `Copy` instance to not copy the parser state by accident in the parser

Reply:
not sure if others have thoughts on it

Reply:
That reasoning makes sense to me, although I do think if we're going to intentionally omit `Copy`, we should leave a comment to that effect in the code!